.dagger/bench.go

```diff
@@ -135,5 +141,37 @@ func (b *Bench) bench(
 	}
 
 	_, err = run(cmd).Sync(ctx)
+
+	return err
+}
+
+func (b *Bench) notifyOnFailure(ctx context.Context, err error, discordWebhook *dagger.Secret) error {
+	if err == nil {
+		return nil
+	}
+	if discordWebhook == nil {
+		return err
+	}
+
+	commit, err := b.Test.Dagger.Git.Head().Commit(ctx)
+	if err != nil {
+		commit = "failed to find commit SHA"
+	}
+
+	daggerCloudURL, err := dag.Notify().DaggerCloudTraceURL(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to fetch trace URL for failed benchmarks: %w", err)
+	}
+
+	message := fmt.Sprintf(
+		"[failed](%s) on SHA [%s](https://github.com/dagger/dagger/commit/%s)",
+		daggerCloudURL,
+		commit,
+		commit,
+	)
+	_, discordErr := dag.Notify().Discord(ctx, discordWebhook, message)
+	if discordErr != nil {
+		return fmt.Errorf("failed to notify discord that benchmarks failed: %w, discord error %s", err, discordErr)
```

Comment:
Should be "%w" for the second error too?

Reply:
weirdly `%w`ing both trips a linter rule, and this doesn't.